docs/documentation/server_admin/topics/clients/client-policies.adoc

```diff
@@ -143,6 +147,8 @@ One of several purposes for this executor is to realize the security requirement
 * Enforce a valid redirect URI that the OAuth 2.1 specification requires
 * Enforce SAML Redirect binding cannot be used or SAML requests and assertions are signed
 
+Another available executor is the `auth-flow-enforce` that can be used to enforce an authentication flow during an authentication request. For instance, it can be used to select a flow based on certain conditions, such as a specific scope or an ACR value. For more details, see the <<_client-poliy-auth-flow, related documentation>>.
```

Comment:
```suggestion
Another available executor is the `auth-flow-enforce`, which can be used to enforce an authentication flow during an authentication request. For instance, it can be used to select a flow based on certain conditions, such as a specific scope or an ACR value. For more details, see the <<_client-poliy-auth-flow, related documentation>>.
```